lib/linguist/languages.yml

```diff
@@ -2708,6 +2708,15 @@ Lean:
   tm_scope: source.lean
   ace_mode: text
   language_id: 197
+Ledger:
+  type: data
+  extensions:
+  - ".dat"
```

Comment:
We've already tried to add support for this extension in the past (cf. #2169), but there are way too many different languages using it. I think it would be best to remove it from this pull request.

Reply:
Hello, I've dropped ledger's auxiliary extensions and kept `.ledger` only. License was also updated.